css/src/atomics/flex.scss

```diff
@@ -0,0 +1,423 @@
+//Align content
+
+.align-content-flex-start {
```

Comment:
I think this is an awesome start! In the past, we've had some issues with the "organic" growth of some atomic style helpers. In order to try and ensure that these names always stay ordered and correct, I think perhaps we should use some loops:

```scss
@use 'sass:map';

$flex-atomics: (
	'justify-content': (
		flex-start,
		flex-end,
		center,
		space-around,
		space-around
	),
	'align-items': (
		flex-start,
		flex-end,
		center,
		baseline,
		stretch
	),
	'align-self': (
		flex-start,
		flex-end,
		center
	),
	'flex-direction': (
		column,
		column-reverse,
		row,
		row-reverse
	),
	'flex-wrap': (
		wrap
	)
) !default;

@each $key in map.keys($flex-atomics) {
	@each $value in map.get($flex-atomics, $key) {
		.#{$key}-#{$value} {
			#{$key}: $value !important;
			@debug 'Should create: #{$key}-#{$value}'; // remove me!
		}
	}
}
```

Reply:
Note that I haven't added the responsive additions here. Let's discuss which ones to add here. Do we really need tablet? Or desktop? Which ones are used most often in the the current code?